cfme/containers/provider/openshift.py

```diff
@@ -25,17 +25,19 @@ def __init__(self, name, value, field_type=None, href=None):
 
 class OpenshiftDefaultEndpoint(ContainersProviderDefaultEndpoint):
     """Represents Openshift default endpoint"""
+
     @staticmethod
-    def get_ca_cert():
+    def get_ca_cert(connection_info):
         """Getting OpenShift's certificate from the master machine.
         Args:
-           No args.
+            connection_info (dict): username, password and hostname for OCP
         returns:
             certificate's content.
         """
-        cert_file_path = path.join(str(data_path), 'cert-auths', 'cmqe-tests-openshift-signer.crt')
-        with open(cert_file_path) as f:
-            return f.read()
+
+        with ssh.SSHClient(**connection_info) as provider_ssh:
```

Comment:
__REQUIRED__
There is already [ssh connection in the provider](https://github.com/ManageIQ/integration_tests/blob/master/cfme/containers/provider/openshift.py#L132), Please insert the provider into the function instead and perform this with cli.run_command (https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/ocp_cli.py#L31). it should be something like:
```python
def get_ca_cert(provider):
    return ''.join(provider.cli.run_command('cat /etc/origin/master/ca.crt')[1].readlines()))
```

Reply:
I tried to implement this kind of solution but OpenShit endpoint doesn't have a provider attribute, and I am not sure that is the right solution for us, due to the size of this object.

I can add an add the provider in [1] instead of ssh_password

[1] https://github.com/shalomnaim1/integration_tests/blob/a232af749a7eee66bb54d245909be5a368623c87/cfme/containers/provider/openshift.py#L206

Reply:
I thought about it, and the solution I suggested here in the previous comment is not valid since ```from_config``` is a class method, so I don't have any access to ```self``` and I don't want to create a dummy provider object to overload ```prov_config```

Reply:
I understand, so this is make sense.